pkg/adapter/config/etcdSensitiveDoguConfigRepository.go

```diff
@@ -47,37 +88,16 @@ func (e EtcdSensitiveDoguConfigRepository) GetAllByKey(ctx context.Context, keys
 	return entries, errors.Join(errs...)
```

Comment:
saveOrDeleteAllByRegistryKeys should just return the error too. The special types return already from the function calls.

Reply:
Good question. My changes are just a refactoring of the config repo branch. Maybe because save and delete only returns internal errors and get can also throw an notfound error?